src/Donations/Components/ContactsForm.tsx

```diff
@@ -290,9 +300,12 @@ function ContactsForm(): ReactElement {
                 />
               )}
             />
-            {errors.email && errors.email.type !== "validate" && (
+            {errors.email && errors.email.type === "required" && (
               <span className={"form-errors"}>{t("emailRequired")}</span>
             )}
+            {errors.email && errors.email.type === "emailInvalid" && (
+              <span className={"form-errors"}>{t("wrongEmailEntered")}</span>
+            )}
```

Comment:
This could be combined. You can check `errors.email.type` to decide the message, as `required` and `emailInvalid` are the only possibilities, or set `errors.email.message` based on the type of error.

Reply:
This also applies for GiftForm and OnBehalf